app/controllers/admin/events_controller.rb

```diff
@@ -37,6 +37,7 @@ def update
   def publish
     @event = find_event
     @event.publish!
+    NewEventMailer.notify_volunteer(@event).deliver
```

Comment:
this can become a background job when you change `deliver` to `deliver_later`. This means that it will move this statement to a pool (like Redis) that can be later on, pulled by a background worker (like Sidekiq) and execute

https://redis.io/
http://sidekiq.org/